moto/s3/responses.py

```diff
@@ -2272,10 +2272,19 @@ def _key_response_post(
             )
 
         elif "restore" in query:
-            es = minidom.parseString(body).getElementsByTagName("Days")
-            days = es[0].childNodes[0].wholeText
+            params = minidom.parseString(body)
```

Comment:
Is there any value in using `xmltodict` here? We use it somewhere else in this file as well.

I don't know what the output for that would look like, but maybe it's easier than parsing the XML manually.